### PR TITLE
refs: Remove unused ScheduleJob model

### PR DESCRIPTION
**Pull request by wedamija**
This is no longer used at all, just removing it.

### CODE REVIEW

File: src/sentry/south_migrations/0480_removed_scheduled_job.py [hunk 1]
```diff
@@ -0,0 +1,1385 @@
...
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = True
```
Comment (wedamija) [inline src/sentry/south_migrations/0480_removed_scheduled_job.py]: @mattrobenolt I assume the best way to do this is to actually let this deploy, and then have #ops remove the table afterwards?

It's currently completely empty, so no data loss concerns.

Comment (markstory) [reply src/sentry/south_migrations/0480_removed_scheduled_job.py]: Using is_dangerous gets around the deploy timing issue, another way would be to deploy the migration separately after the model removal goes out.

Comment (wedamija) [reply src/sentry/south_migrations/0480_removed_scheduled_job.py]: Yeah, maybe I'll just split them up

**Comment by wedamija** [comment]
This has potential use even though it's currently unused, so not removing.

**Pull request closed without merging**